src/hooks/useNotifications.tsx

```diff
@@ -82,24 +97,41 @@ const notificationQuery = gql`
               }
             }
           }
+          ... on PrivatePostNotification {
+            post {
+              id
+              __typename
+              ... on ActivePost {
+                authorId
+              }
+            }
+          }
         }
       }
     }
   }
 `;
 
-const supportedNotificationTypes = [
+export const supportedNotificationsConst = [
   'PostReplyNotification',
   'CircleAdminPostNotification',
   'SurveyAssignedNotification',
-];
+  'PrivatePostNotification',
+] as const;
+
+export type SupportedNotifications = (typeof supportedNotifications)[number];
+
+export const supportedNotifications = supportedNotificationsConst.map(
```

Comment:
Why did we need to do this? Do our typings not include a `__typename` for `PrivatePostNotification`?

Reply:
I updated this with a solution but wondering if I might be missing something more concise/obvious? Let me know what you think.